### PR TITLE
Fixing post request

### PR DESCRIPTION
**Pull request by colinpmacarthur**
@lauraGgit 

Just a couple small tweaks to the POST schema and mappings:
- Updating the POST mapping to note the middleware should store the files
- Updating the schema file to match the mapping file. They had fallen out of sync.

### CODE REVIEW

File: special-use/special-use-data-schema.json [hunk 1]
```diff
@@ -1,64 +1,87 @@
 {
     "$schema": "http://json-schema.org/draft-04/schema#",
     "title": "Special use application",
-    "description": "An application for a special use permit from the U.S. Forest Service",
+    "description": "A rough sketch of the POST JSON HTTP request for a special use permit from our ePermitting system the U.S. Forest Service",
```
Comment (lauraGgit) [inline special-use/special-use-data-schema.json]: `from our ePermitting system the U.S. Forest Service` maybe the `from the U.S. Forest Service ePermitting System`.

File: special-use/special-use-data-schema.json [hunk 2]
```diff
@@ -1,64 +1,87 @@
...
+          "mailingCity": { "type": "string"},
+          "mailingState": { "type": "string"},
+          "mailingZIP": { "type": "integer"},
       		"organizationName": {
       			"description": "The name of the group, company, organization or event applying",
       			"type": "integer",
       		},
-          "website": { "type": "string" },
-        "required": ["firstName","lastName","dayPhone","emailAddress","mailingAddress","mailingCity","mailingZIP","mailingState","organizationName"]
+          "website": { "type": "string", "suds-table": "Contact" },
```
Comment (lauraGgit) [inline special-use/special-use-data-schema.json]: Need to note that this is not part of the SUDS contact, but is standalone.

File: special-use/special-use-data-schema.json [hunk 3]
```diff
@@ -1,64 +1,87 @@
...
, "Nonprofit" ] }},
+        "required": ["firstName","lastName","dayPhone","emailAddress","mailingAddress","mailingCity","mailingZIP","mailingState"]
+      },
+      "phoneNumber": {
+        "type": "object",
+        "properties": {
+          "areaCode": { "type": "integer"},
+          "number": { "type": "integer"},
+          "extension": { "type": "integer"},
+          "type": { "basic-api-field" : "PHONE_NUMBER_TYPE"},
```
Comment (lauraGgit) [inline special-use/special-use-data-schema.json]: Are there options for this? I.e. Business, etc.

File: special-use/special-use-data-schema.json (hunk 1 above)
Comment (colinpmacarthur) [reply special-use/special-use-data-schema.json]: Fixed

File: special-use/special-use-data-schema.json (hunk 3 above)
Comment (colinpmacarthur) [reply special-use/special-use-data-schema.json]: Fixed

**Review by lauraGgit — APPROVED** [review]

**Pull request merged**